provisioning/transport/http/devdoc/http_requirements.md

```diff
@@ -43,10 +43,14 @@ Queries the dps to obtain a buffer that may be used as a symmetric key to perfor
 
 **SRS_NODE_PROVISIONING_HTTP_18_009: [** `registrationRequest` shall PUT the registration request to 'https://{provisioningHost}/{idScope}/registrations/{registrationId}/register' **]**
 
+**SRS_NODE_PROVISIONING_HTTP_06_012: [** If the `registrationRequest` response contains a status code >= 429, the result.status value will be set with `registering` and the callback will be invoked with *no* error object. **]**
+
```

Comment:
and now we have to handle this in all transports instead of consistently across transport :-/

Reply:
I'm sorry, I completely miss what this statement is about.  Again, this provokes zero new work on the  parts of the transports.  There has ALWAYS been a hole in this SDK and every other SDK wrt handling a failed registration request.  This PR addresses this with extreme consistency with practically zero change utilizing the already well tested design principles of the polling machine and the transports.  It quite simply could NOT have fit in better with the entire design philosophy that was approved 15 months ago.

---
In reply to: [282200167](https://github.com/Azure/azure-iot-sdk-node/pull/555#discussion_r282200167) [](ancestors = 282200167)